### DISCUSSION TITLE
Remove path info from skip exception

### DISCUSSION THREAD

**Discussion by bybilo78** [pytest-dev/pytest#12911] [General]
Given are the following to skip a test.

![image](https://github.com/user-attachments/assets/788bb936-061e-4aef-9da6-b325b64f10c0)

The log result is as follows.
![image](https://github.com/user-attachments/assets/71c8472b-175b-49c0-ac6c-620203324b6b)

The problem is, that the same tests are running on different os with different pytest project paths.
Is a hook or something else to remove the path info in the skip exception?

**Comment by RonnyPfannschmidt** [comment]
I recommend turning that intentionally absolute path and turning it into a path relative to the cwd

**Reply by bybilo78** [reply]
Can you give me an example of how to configure pytest to work with the relative path?

**Comment by dongfangtianyu** [comment]
Currently, there isn't a setup like this, but we could consider creating an issue or PR for it.
Are you using junitxml or any plugins? By default, this information won't be included in the logs
